help/license/reuse.md

```diff
@@ -0,0 +1,49 @@
+Reuse Requests
+==============
+
+This FAQ is an attempt to collect answers to your common questions surrounding reusing content from arXiv in your materials. 
+
+- [Can I reuse figures from an arXiv paper?](#figures) 
+- [Do I need arXiv's permission to repost the full text?](#full_text)
+- [How can I determine what license the version was assigned?](#what_license)
+- [I want to include a paper of mine from arXiv in my thesis, do I need specific permission?](#thesis)
+- [I want to include a paper of mine from arXiv in an institutional repository, do I need permission?](#ir)
+- [Can I harvest the full text of works?](#bulk_data)
+
+<span id="figures"></span>
+## Can I reuse figures from an arXiv paper?
+
+The short answer is "it depends". More specifically: 
+  - If the [license](/help/license) applied to the work allows for remixing or reuse with citation, then yes. 
+  - If not, then the version is assigned one of the [arXiv perpetual non-exclusive licenses](https://arxiv.org/licenses/nonexclusive-distrib/1.0/), and you will need to contact the submitter or copyright holder (if published) to determine applicable permissions. 
+
+<span id="full_text"></span>
+## Do I need arXiv's permission to repost the full text?
+
+**Note:** All e-prints submitted to arXiv are subject to copyright protections. arXiv is not the copyright holder on any of the e-prints in our corpus. 
+
+In some cases, submitters have provided permission in advance by submitting their e-print under a permissive [Creative Commons license](/help/license#licenses-available). The overwhelming majority of e-prints are submitted using the [arXiv perpetual non-exclusive license](https://arxiv.org/licenses/nonexclusive-distrib/1.0/), which does not grant further reuse permissions directly. In these cases you will need to contact the author directly with your request.
+
+<span id="what_license"></span>
+## How can I determine what license the version was assigned?
+
+All arXiv abstract pages indicate a [license](/help/license) underneath the "Download:" options. 
```

Comment:
Is it possible to change this link from just the word 'license'? This page already has that single word as a link so it will be seen as duplicate. What about 'assigned license' here?

Reply:
Sure.